portals/publisher/source/src/app/components/Apis/Details/Endpoints/EndpointOverview.jsx

```diff
@@ -242,9 +252,36 @@ function EndpointOverview(props) {
         return supportedEndpointTypes;
     };
 
+    /**
+     * Retrieve the service list from the service catalog
+     */
+    function getServices() {
+        const promisedServices = ServiceCatalog.getServiceList();
+        promisedServices.then((response) => {
+            setServicesList(response.list);
+
+        }).catch((error) => {
+            if (error.response) {
+                Alert.error(error.response.body.description);
+            } else {
+                Alert.error(
+                    <FormattedMessage
+                        id='Apis.Details.APIDefinition.Addservice.service.retrieve.error'
+                        defaultMessage='Something went wrong while retrieving the services'
+                    />,
+                );
+            }
+            console.error(error);
+        }).finally(() => {
```

Comment:
Why we need an empty finally here ?

Reply:
Removed from https://github.com/wso2/apim-apps/pull/144/commits/a9f8d7cc1b607b689b3bfd323dbb451de0c6d78c